nat-lab/tests/telio.py

```diff
@@ -398,6 +400,7 @@ async def on_stdout(stdout: str) -> None:
         self._runtime = Runtime()
         self._events = Events(self._runtime)
         self._router = new_router(self._connection)
+        self._router.ip_stack = self._ip_stack
```

Comment:
Currently router initializes its IP stack to some default value, which is never used. I think that removing this setter and moving the `ip_stack` field to the constructor would be much easier to read.

Reply:
FIxed